unittests/test_transform.cpp

```diff
@@ -293,7 +293,7 @@ void parallel_transform3(size_t W) {
     /** Target implementation with Subflow::transform */
     std::vector<string> tgt;
 
-    std::vector<string>::iterator tgt_beg;
+    typename std::vector<string>::iterator tgt_beg;
```

Comment:
I'm not really sure about this change.  However without it I was getting the error

```
Microsoft Windows [Version 10.0.22621.1555]
(c) Microsoft Corporation. All rights reserved.

D:\checkouts\taskflow\build>C:\PROGRA~2\MICROS~3\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx86\x86\cl.exe  /nologo /TP -ID:\checkouts\taskflow -ID:\checkouts\taskflow\3rd-party\doctest /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -std:c++17 -MDd   /bigobj /W3 /permissive- /Founittests\CMakeFiles\test_transform.dir\test_transform.cpp.obj /FdTARGET_COMPILE_PDB /FS -c D:\checkouts\taskflow\unittests\test_transform.cpp
test_transform.cpp
D:\checkouts\taskflow\taskflow\core\observer.hpp(847): warning C4244: 'initializing': conversion from '_Rep' to 'size_t', possible loss of data
        with
        [
            _Rep=__int64
        ]
D:\checkouts\taskflow\taskflow\core\observer.hpp(884): warning C4244: '=': conversion from '_Rep' to 'size_t', possible loss of data
        with
        [
            _Rep=__int64
        ]
D:\checkouts\taskflow\unittests\test_transform.cpp(296): error C2760: syntax error: unexpected token 'identifier', expected ';'
D:\checkouts\taskflow\unittests\test_transform.cpp(296): error C7510: 'iterator': use of dependent type name must be prefixed with 'typename'

D:\checkouts\taskflow\build>

```